sh_script/build.sh

```diff
@@ -203,16 +204,22 @@ run() {
 CHECK_OPTION=false
 BUILD_OPTION=false
 RUN_OPTION=false
+TARGET_OPTION=x86_64-unknown-none
```

Comment:
Unsure if it's bad practice in shell scripts to define the variable later than a function may refer to it